server/routes.js

```diff
@@ -244,6 +244,7 @@ export default async app => {
   const graphqlServerOptions = {
     introspection: true,
     playground: isDevelopment,
+    csrfPrevention: true,
```

Comment:
I believe it does not because we are using POST requests and there was no multipart request up to now.

> If you enable this feature, any clients that send operations via GET requests or multipart upload requests must include a non-empty Apollo-Require-Preflight (or any other non-special-cased) header in that request

https://www.apollographql.com/docs/apollo-server/v3/security/cors/